tests/cases/standalone/common/alter/modify_col.result

```diff
@@ -1,39 +1,49 @@
-CREATE TABLE test(i INTEGER, j TIMESTAMP TIME INDEX);
+CREATE TABLE test(id INTEGER PRIMARY KEY, i INTEGER, j TIMESTAMP TIME INDEX, k BOOLEAN);
 
 Affected Rows: 0
 
-INSERT INTO test VALUES (1, 1), (2, 2);
+INSERT INTO test VALUES (1, 1, 1, false), (2, 2, 2, true);
 
 Affected Rows: 2
 
 ALTER TABLE test ALTER COLUMN "I" TYPE STRING;
 
 Error: 4002(TableColumnNotFound), Column I not exists in table test
 
-ALTER TABLE test ALTER COLUMN I TYPE STRING;
+ALTER TABLE test ALTER COLUMN k TYPE DATE;
 
-Affected Rows: 0
+Error: 1004(InvalidArguments), Invalid alter table(test) request: column 'k' cannot be cast automatically to type 'Date'
 
-SELECT * FROM test;
+ALTER TABLE test ALTER COLUMN id TYPE STRING;
 
-+---+-------------------------+
-| i | j                       |
-+---+-------------------------+
-| 1 | 1970-01-01T00:00:00.001 |
-| 2 | 1970-01-01T00:00:00.002 |
-+---+-------------------------+
+Error: 1004(InvalidArguments), Not allowed to modify index column id from table test
```

Comment:
nit:
```suggestion
Error: 1004(InvalidArguments), Not allowed to modify index column `id` of table test
```

Reply:
as you wish

but single quotes are not used elsewhere (like `add column`)